### PR TITLE
Font wizard modification

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `font-wizard.py` updated

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR 💯 
Test it on [fsymbols.com](https://fsymbols.com/)

File: otherfile/font_wizard.py [hunk 1]
```diff
@@ -102,8 +102,11 @@ def is_ascii(s):
             print(Error2)
             sys.exit()
     font_dic = dict(zip(Letters, font_data))
+    font_split_list = list(map(lambda x: len(x.split("\n")), font_dic.values()))
+    font_height = font_split_list[0]
```
Comment (sadrasabouri) [inline otherfile/font_wizard.py:106]: Shouldn't it be `font_height = len(font_split_list[0])`?

Comment (sepandhaghighi) [reply otherfile/font_wizard.py]: No, but I didn't choose a proper name for variable names.
Fixed in https://github.com/sepandhaghighi/art/pull/185/commits/9f01cc4c984e141bfa20e94f9688b6c253e7f851

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Please update the `CHANGELOG.md`

**Comment by sepandhaghighi** [comment]
> Please update the `CHANGELOG.md`

https://github.com/sepandhaghighi/art/blob/9f01cc4c984e141bfa20e94f9688b6c253e7f851/CHANGELOG.md?plain=1#L10

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**